### PR TITLE
Sb static pages/default login page preset2

### PR DESCRIPTION
**Pull request by Tim-M-S**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: src/Widgets/Presets/DefaultLoginPreset.php [hunk 1]
```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Ceres\Widgets\Presets;
+
+use Ceres\Config\CeresConfig;
```
Comment (cjohannsen) [inline src/Widgets/Presets/DefaultLoginPreset.php]: remove usage

**Review by cjohannsen — CHANGES_REQUESTED** [review]

File: resources/lang/de/Template.properties [hunk 2]
```diff
@@ -340,6 +340,7 @@ liveShoppingSeconds = "Sekunden"
 login = "Anmelden"
 loginBackToLogin = "Zurück"
 loginBlocked = "Ihr Kundenkonto wurde gesperrt. Bitte kontaktieren sie den Betreiber."
+loginCallToAction = "Noch kein Kunde?"
```
Comment (fmutschler) [inline resources/lang/de/Template.properties]:
```suggestion
loginCallToAction = "Sie sind noch kein Kunde?"
```

File: src/Widgets/Presets/DefaultLoginPreset.php (hunk 1 above)
Comment (Tim-M-S) [reply src/Widgets/Presets/DefaultLoginPreset.php]: done

File: resources/lang/en/Template.properties [hunk 3]
```diff
@@ -340,6 +340,7 @@ liveShoppingSeconds = "Seconds"
 login = "Log in"
 loginBackToLogin = "Back"
 loginBlocked = "Your account has been blocked. Please contact the provided."
+loginCallToAction = "Not a customer yet?"
```
Comment (fmutschler) [inline resources/lang/en/Template.properties]:
```suggestion
loginCallToAction = "You are not a customer yet?"
```

**Review by fmutschler — CHANGES_REQUESTED** [review]

**Review by fmutschler — APPROVED** [review]

**Review by cjohannsen — APPROVED** [review]

**Pull request merged**